src/config.js

```diff
@@ -44,6 +44,8 @@ const defaultConfig = {
     )} use ${chalk.yellow(
       "`npm run test:updateCoverageExceptions`",
     )} to update the coverage threshold for these files.`,
+    // Override this message with a custom documentation prompt.
+    documentationPrompt: undefined,
```

Comment:
can you adjust this to be a sample documentation prompt (and then you won't need the comment), and then add a separate config under the `output` section for whether to display the documentation prompt (defaulted to false for backwards compatibility)?

Reply:
Done!

Reply:
@jcw- the only downside of doing it this way means you have to override all of the `output` variables in the config:

```
  "output": {
    "generatedCoverageExceptionsPath": "generatedCoverageExceptions.json",
    "additionalReports": {
      "html": "coverage/html-report"
    },
    "displayDocumentationPrompt": true
  }
```

Reply:
thanks! ah, good point about that side effect - i'll have to think about a better config merging strategy so that won't be required

Reply:
or maybe someone will come along and beat me to it! sweet, thanks @seanhealy
https://github.com/GetJobber/jest-a-coverage-slip-detector/pull/36